kartotherian/Dockerfile

```diff
@@ -20,18 +20,10 @@ RUN apt update && \
     apt-get install -y nodejs
 
 
-RUN git clone https://bitbucket.org/qmap/demultiplexer.git /opt/demultiplexer
```

Comment:
we don't need demultiplexer and snapshot anymore ?

Reply:
They are still required, but these custom versions are now directly referenced in our `package.json`.